src/app/public/src/modules/help-directive/disable.directive.ts

```diff
@@ -0,0 +1,19 @@
+import { Directive, OnInit, OnDestroy } from '@angular/core';
+
+import { HelpWidgetService } from '../shared';
+
+@Directive({
+  selector: '[disableWidget]'
```

Comment:
The directive should be prefixed with “help” or “bbHelp” so that if it IS used elsewhere, people know where it belongs. https://angular.io/guide/styleguide#custom-prefix-for-directives